demos/simple/demo.js

```diff
@@ -7,8 +7,9 @@ import {createMap} from './osm';
 import {Overlay} from "ol";
 import {unByKey} from 'ol/Observable';
 import '@geoblocks/elevation-profile';
+import SnappedDensifier from '../../src/densifier/SnappedDensifier'
 
-const ROUTING_URL = 'https://graphhopper-all.schweizmobil.ch/route?vehicle=schmwander&type=json&weighting=fastest&elevation=true&way_point_max_distance=0&instructions=false&points_encoded=true';
+const ROUTING_URL = 'https://graphhopper-all.schweizmobil.ch/route?profile=schmwander&type=json&elevation=true&way_point_max_distance=0&instructions=false&points_encoded=true&details=surface&details=hiking_category&details=structure';
```

Comment:
The `details` are something custom to CHM, they are not needed here and would cause problems

Reply:
I removed it, but it was working the same because we use the same image here